src/app/docs/timepicker/timepicker-docs.component.html

```diff
@@ -221,4 +236,15 @@
 
   </sky-docs-design-guidelines>
 
+  <sky-docs-code-examples
+    [packageDependencies]="{
+      '@skyux/datetime': '*'
+    }"
+  >
+    <sky-docs-code-example
+      heading="Basic setup"
```

Comment:
How would you feel about not using directories and headings for `basic` if we only show one example? I've been trying to keep that consistent as we work across demos.

Reply:
You okay if I keep the directory, but remove the heading? My reasoning: if we need to add another code example in the future, the directory structure won't need to be changed and the "basic" files won't show up in the new pull request.

Reply:
Yup that works for me! I was more worried about presentation.